### ISSUE TITLE
Problem with Document List Panel Extension column

### ISSUE THREAD

**Issue by 123dev** [notepad-plus-plus/notepad-plus-plus#10363]
### Description of the Issue
Document List Panel's Extension column is right justified and clipped, it makes it impossible to know an extension which is more than 2 characters long.

![image](https://user-images.githubusercontent.com/13109154/128766142-c425ac3b-3726-44f7-97bf-3a789e4afe79.png)

When the columns are resized to address the issue
![image](https://user-images.githubusercontent.com/13109154/128766233-54d29d1e-f112-431a-b449-770a9602ae74.png)

All is good until you switch away from Npp (example minimize / maximize)
And it's back to its original clipped state.

### Steps to Reproduce the Issue
1. Enable Document List panel and show the extensions 
2. Extension column is clipped
3. Resize column to fix it
4. Minimize Npp
5. Maximize Npp
6. Problem is back

### Expected Behavior
I expect the columns not to be clipped as they are by default (at least allow more characters than 2)
And if one resizes the columns, I expect them to be persistent until user resizes again, ie it survives restarts let alone minimize / maximize.

### Actual Behavior
See Description and Screenshots above

### Debug Information
Notepad++ v8.1.2   (32-bit)
Build time : Jul 16 2021 - 01:56:21
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19043.1151
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll DSpellCheck.dll Explorer.dll JSMinNPP.dll LocationNavigate.dll MarkdownViewerPlusPlus.dll mimeTools.dll NppConverter.dll NppExec.dll NppExport.dll NPPJSONViewer.dll NppSaveAsAdmin.dll NppSnippets.dll NppTextFX.dll Tidy2.dll _CustomizeToolbar.dll

**Comment by ivan-u7n** [comment]
I can't reproduce it using
```
Notepad++ v8.1.2   (64-bit)
Build time : Aug  8 2021 - 15:24:20
Path : F:\develop\notepad-plus-plus-master\PowerEditor\bin\NotepadPP-release.exe
Command Line : -multiInst
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 10 Enterprise LTSC 2019 (64-bit) 
OS Version : 1809
OS Build : 17763.2061
Current ANSI codepage : 1251
Plugins : none
```

Have you tried running Notepad++ without plugins?

**Comment by ArkadiuszMichalski** [comment]
I can confirm this bug. It would also be nice to remember the sizes of these columns in `config.xml`, because setting the size of the columns each time when the program is run is annoying.

**Comment by ivan-u7n** [comment]
Oh, I now see it. It seems the width of the Ext. column is set to some fixed value that is good enough for standard 3-letter extensions in standard font size.

@123dev Do you by chance use a HiDPI monitor, or enlarge system font metrics in some other way?

**Comment by donho** [comment]
@123dev 
it seems you're using a high dpi display. The ext width in high dpi display has been fixed in v8.1.4.
